changes/8782.outlier_detection.rst

```diff
@@ -0,0 +1,3 @@
+Decrease the amount of file I/O required to compute the median when in_memory is set to False.
```

Comment:
Would you split these 3 entries into 3 files `8782.outlier_detection.0.rst` `8782.outlier_detection.1.rst` `8782.outlier_detection.2.rst`.? This helps with the eventual towncrier formatting.